ansible/pbTestScripts/testScript.sh

```diff
@@ -194,6 +195,9 @@ startVMPlaybookWin()
 		cd $WORKSPACE/adoptopenjdkPBTests/$folderName-$branchName/ansible
 	fi
 	ln -sf Vagrantfile.$OS Vagrantfile
+	# Remove the Hosts files if they're found
+	[[ -f playbooks/AdoptOpenJDK_Windows_Playbook/hosts.tmp ]] && rm -f playbooks/AdoptOpenJDK_Windows_Playbook/hosts.tmp
```

Comment:
Get rid of the check on these two lines as it's not needed when using `rm -f`

Reply:
Resolved with latest commit